### DISCUSSION TITLE
AlmaLinux OS 9

### DISCUSSION THREAD

**Discussion by Nirjonadda** [netdata/netdata#15471] [Q&A] [answered]
Hello,

AlmaLinux 9 are supported by Netdata ?

**Comment by thiagoftsm** [comment, marked as answer]
Hello @Nirjonadda ,

I have tested different times `netdata` on `Alma 9`, but I tested it compiling `netdata` from scratch.  We also inform about support for it according to our [documentation](https://learn.netdata.cloud/docs/installing/platform-support-policy).

A simple way for you install `netdata` on your host/server is running our [script](https://learn.netdata.cloud/docs/installing/one-line-installer-for-all-linux-systems).

Please, let us know if you still have any doubt!

Best regards!

**Comment by Nirjonadda** [comment]
Also does updated GPG-KEY because SHA-1 deprecation on AlmaLinux 9

See here: https://www.redhat.com/en/blog/rhel-security-sha-1-package-signatures-distrusted-rhel-9

**Reply by thiagoftsm** [reply]
Thank you for your update @Nirjonadda !

@netdata/agent-sre , please, can you take a look on this for us?

**Reply by Ferroin** [reply]
I’ve updated the signing config so that newly published packages should have SHA256 signatures.

**Comment by Nirjonadda** [comment]
How to check this SHA1 or SHA256 signatures via SSH Command?
